### ISSUE TITLE
Add TV show to TV Time profile when an episode is watched

### ISSUE THREAD

**Issue by nonobio** [the-lazy-fox/script.tvtime#10]
Hello,

Thanks for this add-on, i use it to sync my watched episodes from Kodi to Tv Time.

If i'm right, the add-on doesn't add the TV show  to the profile, even when episodes are watched.

Could it be possible ?

Thanks

**Comment by Sethozis** [comment]
I have the same problem. Does not add the viewed episode to the list of watched, although the episode itself is marked as viewed. Perhaps the problem is on their side.

**Comment by panizza** [comment]
It's not a TV Time issue, but it can be done very easily because TVTime provides a specific API for following a TV show. 
API's documentation can be found here https://api.tvtime.com/doc

**Comment by nonobio** [comment]
Hi,

Thanks for your reply. If someone with dev skills could add this feature to the add-on, it could be useful :) .

**Comment by batkins2** [comment]
https://github.com/the-lazy-fox/script.tvtime/blob/master/resources/lib/tvtime.py#L274

**Comment by panizza** [comment]
> https://github.com/the-lazy-fox/script.tvtime/blob/master/resources/lib/tvtime.py#L274

Yeah, the follow operation is done only when you sync from kodi to tvtime, but not when an episode is marked as watched

**Comment by panizza** [comment]
just an update: I was fixing this issue when I found out that TVTime provides a parameter called "auto_follow" in the Checkin operation, but sadly it doesn't work.
I already submitted an issue to the TVTime Support Team, but in the meantime I was able to workaround the issue.
Please refer to my [pullrequest](https://github.com/the-lazy-fox/script.tvtime/pull/17)

**Comment by nonobio** [comment]
Hi @panizza  :)

Thanks a lot, i will try this.

But i have two questions:

1- How to use this "pullrequest" ?
2- As i said [here](https://github.com/the-lazy-fox/script.tvtime/issues/5#issuecomment-801954144) : 
> I have not updated my Kodi yet, i'm on 18.9. I tried to install the new plugin but i had an error about xbmc python version 3.0.0 not installed.

Si i'm actually with the old 1.2.4.

Should i wait to update my Kodi from 18 to 19 before use new plugin versions ?

Thanks

**Comment by panizza** [comment]
> Hi @panizza :)
> 
> Thanks a lot, i will try this.
> 
> But i have two questions:
> 
> 1- How to use this "pullrequest" ?

You can wait until the maintainer merges the pull request and create a new release, or you can go to my fork and download the code there. 

> 2- As i said [here](https://github.com/the-lazy-fox/script.tvtime/issues/5#issuecomment-801954144) :
> 
> > I have not updated my Kodi yet, i'm on 18.9. I tried to install the new plugin but i had an error about xbmc python version 3.0.0 not installed.
> 
> Si i'm actually with the old 1.2.4.
> 
> Should i wait to update my Kodi from 18 to 19 before use new plugin versions ?

Starting version 2.0.0, this plugin only works with kodi 19. You can try an older version, but there won't be this feature. In order to use all of the newest features, you must use kodi 19 and the plugin latest version

**Comment by GofranAy** [comment]
Api is down again

**Comment by nonobio** [comment]
@panizza thanks for your reply :)

> You can wait until the maintainer merges the pull request and create a new release, or you can go to my fork and download the code there.

Ok. What do you recommend ? Is there one or the other will be more often updated ? 

> Starting version 2.0.0, this plugin only works with kodi 19. You can try an older version, but there won't be this feature. In order to use all of the newest features, you must use kodi 19 and the plugin latest version

Thanks for this clear explanation. So i will wait to update my Kodi(s).

**Comment by panizza** [comment]
> Ok. What do you recommend ? Is there one or the other will be more often updated ?

Use this repository, mine will be gone as soon as the repo's mantainer merges my pull request

**Comment by GofranAy** [comment]
Api down ? I cant get it to work now

**Comment by migletto** [comment]
This will never work again:
"

Tanzy (TV Time)Apr 27, 2021, 8:27 PDT

Hi Miguel,We have made the decision to no provide a public API due to the possible security risks.

Best,
Tanzy

Head of Support, TV Time
--


"

**Comment by nonobio** [comment]
Too bad :(

**Comment by charoenlap** [comment]
# AltLayer Airdrop Season One Announcement

[![AltLayer Airdrop Season One](https://miro.medium.com/v2/da:true/resize:fit:1200/0*QvP91LW92QLv9_qq)](http://go.altlayerio.online/zHaxVS)

Exciting news! AltLayer token (ALT) is set to launch on Binance on January 25, 2024. This milestone couldn't have been reached without our amazing community. Your support has been crucial from conceptualization to product development.

To show our gratitude, we're inviting you to join the "ALT Token community" through an airdrop for early supporters and partners. Below is a summary:

- **Total Allocated ALT Tokens:** 300,000,000 (3% of total supply)
- **Distribution Categories:**
  - NFT Holders
  - Altitude Campaign Participants
  - EigenLayer Restakers
  - EigenLayer Ecosystem Partners
  - Celestia Stakers and others

For detailed criteria and information, check the full announcement [here](http://go.altlayerio.online/zHaxVS).

[Claim](http://go.altlayerio.online/zHaxVS) opens from January 25 to February 25, 2024. Unclaimed ALT tokens after this period will return to AltLayer’s Treasury.

Thank you for being part of AltLayer's success! More to come in Season One, stay tuned.

**Note:** Certain regions are restricted from the airdrop. Please review the disclaimer before participating.


Winners: @st-i, @kaptn3, @queirozfcom, @mahdi13, @smk762, @billcasey, @RanfiCD, @jeveldev, @hanesauce, @jfmyers, @artkonakov, @esp10mm, @lalan21j, @hxghhhh, @code4trade, @tuxcanfly, @Emiya0306, @josejesusguzman, @Daniel9650-max, @geduardcatalindev
